Main.elm

```diff
@@ -1,7 +1,288 @@
-module Main exposing (..)
+port module Main exposing (..)
 
 import Html
+import Html.Attributes
+import Html.Events
+import Json.Decode as Decode
+import Json.Encode as Encode
+import Kinto
+
+
+-- Settings
+-- If you change the kintoServer there you want to change it in ports.js too.
```

Comment:
Shouldn't it be passed as an app flag then? Or maybe keep it here and pass it as a port call param instead. It would be good to have app settings defined in a single place.